schematic/schemas/json_schema_component_generator.py

```diff
@@ -243,10 +239,10 @@ def get_component_json_schema(
         self,
     ) -> None:
```

Comment:
I usually have an aversion to have a method with the name `get_...` that doesn't return anything. It may not make sense at this point to go and modify this, but I would say that we should keep it in mind for future iterations.

It's find that a class method updates values on the class, but I would also expect that it returns a value so you could easily do:

```
the_value_want = my_class_instance.get_method_to_do_thing()
```

Reply:
@BryanFauble @thomasyu888 There's enough interest in turning JSONSchemaComponentGenerator into a function. I'll create a ticket for doing so.

Reply:
https://sagebionetworks.jira.com/browse/SCHEMATIC-302